geomesa-blobstore/geomesa-blobstore-core/src/main/scala/org/locationtech/geomesa/blob/core/AccumuloBlobStore.scala

```diff
@@ -45,29 +48,30 @@ class AccumuloBlobStore(ds: AccumuloDataStore) extends LazyLogging with BlobStor
   val bw = connector.createBatchWriter(blobTableName, bwc)
```

Comment:
we should close this batch writer during cleanup somehow... add a ticket at least